### PR TITLE
update trunk to have readme, composer and travis

### PR DESCRIPTION
**Pull request by Flyingmana**
also I request to change the default branch to the actual trunk (or take the chance to introdue a master branch)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# composer related
+/composer.lock
+/composer.phar
```
Comment (sanmai) [inline .gitignore]: This has to go into the global gitignore, not repository-local.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+language: php
+php:
+- 5.3
```
Comment (sanmai) [inline .travis.yml]: "PHP 5.3 is supported only on Precise." - it may be easier just to remove

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  ],
+  "autoload": {
+    "psr-0": {
+      "HTML": "./"
+    }
+  },
+  "include-path": [
+    "./"
+  ],
+  "license": "PHP License",
```
Comment (sanmai) [inline composer.json]:
```suggestion
  "license": "PHP-3.0",
```

**Review by sanmai — CHANGES_REQUESTED** [review]

File: composer.json (hunk 3 above)
Comment (sanmai) [reply composer.json]:
```
$ composer validate
./composer.json is valid, but with a few warnings
License "PHP License" is not a valid SPDX license identifier, see https://spdx.org/licenses/ if you use an open license.
```

File: .gitignore (hunk 1 above)
Comment (Flyingmana) [reply .gitignore]: I took this by example from another pear repository and thought some maintainers may have an easier live with this. But I also dont think this has to be part of the repository

File: .travis.yml (hunk 2 above)
Comment (Flyingmana) [reply .travis.yml]: yeah, I wonder if I should maybe keep only 5.6 from the last major version in general

Comment (sanmai) [reply .travis.yml]: I'd keep only 5.6 and up.

File: .gitignore (hunk 1 above)
Comment (sanmai) [reply .gitignore]: Yeah, I think this was a mistake on their part. And maybe several years ago it was OKish.

Comment (sanmai) [reply .gitignore]: Ah, it seems GitHub adds composer.phar by default even these days. E.g. try creating a new repository and select .gitignore template. Bummer!

I'd leave only:
```
/composer.lock
/vendor
```

File: composer.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "pear/html_form",
+  "description": "More info available on: http://pear.php.net/package/HTML_Form",
+  "type": "library",
+  "authors": [
+  ],
```
Comment (sanmai) [inline composer.json:16]: package.xml says it must be `Stig Sæther Bakken <stig@php.net>`

Comment (Flyingmana) [reply composer.json:16]: thanks, did not think about looking there, have added the two maintainers from there

**Review by sanmai — APPROVED** [review]
LGTM.

**Review by ashnazg — APPROVED** [review]

**Comment by ashnazg** [comment]
@Flyingmana , sorry this took me so long to get to.  I have it merged, I have the package on packagist, and have tagged most of the releases.  Because none of the older releases contained composer.json, none of those releases will show up on packagist.  This v1.3.1 that I just released will, though.

**Comment by Flyingmana** [comment]
Thank you a lot, the newest version is probably the most important one :)

**Pull request merged**